### ISSUE TITLE
payu collate -i K collates outputK but restartK-1

### ISSUE THREAD

**Issue by aekiss** [payu-org/payu#275]
`payu collate -i K` collates collates outputK but restartK-1. 
Is this the intended behaviour? If so, it could be helpful to explain this in https://payu.readthedocs.io/en/latest/usage.html#postprocessing

**Comment by aidanheerdegen** [comment]
Yes it is the intended behaviour. This means the current tiled restarts are used in the next submission, which should be faster.

Additionally it would not be possible to safely start the next run due to the possibility the restarts it would require are in some "half collated" state.

I you need to collate a specific restart dir you can use the `-d` option. 

It is documented here:

https://payu.readthedocs.io/en/latest/config.html#collation

```yaml
restart (Defaut: False)
   Collate restart files from previous run.
```

It isn't on by default, so I don't think it's strictly necessary in the usage section, but if you think it would help feel free to pop in a PR.

**Comment by aekiss** [comment]
OK thanks, that makes sense. I've enabled restart collation in all the new access-om2 configs (since it's so fast with mppnccombine-fast) so I've made a PR which (I think!) makes this behaviour a little clearer.

**Comment by aidanheerdegen** [comment]
FYI if you put "Fixes #275" in the PR it will auto close the issue.
